values.yaml

```diff
@@ -7,22 +7,24 @@
 ## If you are not sure, find it like this: `grep cluster-cidr /etc/kubernetes/manifests/kube-controller-manager.yaml | cut -d"=" -f2`
 ## PS: if you had flannel before, your clusters will have "10.244.0.0/16"
 
-podCIDR: ""
-#podCIDR: "192.168.0.0/16"
+podCIDR: "10.42.0.0/16"
+podIP4BlockSize: 24
+podCIDR6: "fd42::/48"
+podIP6BlockSize: 118
 flexVolumePluginDir: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec"
 vethMtu: 0
 autoHostEndpoints: false
 
 typha:
   image:
-    registry: "docker.io"
+    registry: "docker.io/rancher"
```

Comment:
I'd like to use official images when possible.

Reply:
I just want to create a PR to enable my RKE2 calico deployement dual-stack in my local repo.  Thanks to look into this and sorry for the bothering. I will close this PR.